### ISSUE TITLE
"eslint.alwaysShowStatus": true not working anymore after vscode upgrade

### ISSUE THREAD

**Issue by tohbansoon** [microsoft/vscode-eslint#848]
Here is my vscode setting, but after upgrade vscode to the latest version.. my eslint do not show the warning or error status bar anymore...

VSCode Settings:
```
{
  "terminal.integrated.shell.osx": "/bin/zsh",
  "window.zoomLevel": 0,
  "editor.formatOnSave": true,
  "diffEditor.ignoreTrimWhitespace": false,
  "breadcrumbs.enabled": true,
  "eslint.alwaysShowStatus": true,
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true,
  }
}
```

VSCode Version:
```
Version: 1.41.0
Commit: 9579eda04fdb3a9bba2750f15193e5fafe16b959
Date: 2019-12-11T17:58:38.338Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 18.7.0
```

**Comment by dbaeumer** [comment]
@tohbansoon parts of this is by design now. If the loading of the eslint library fails I only show an error and update the status if the `eslint.validate` setting contains the langauge id for which you want to enforce and check validation. In all other cases the extension now stays silent.

Can you provide me with a more concrete description in which case you expect the status to show.

**Comment by tohbansoon** [comment]
sorry that it's not related to vscode version... I downgrade my eslint extension to `1.9.1` and it's working now

**Comment by Jer-X** [comment]
@dbaeumer there is a question want to ask you for answers.

If my opening workspace like above photo

**Comment by dbaeumer** [comment]
@tohbansoon I would like to understand what part of the feature you are missing in ESLint 2.0.x in regards to the status?
